spec/mailers/previews/candidate_mailer_preview.rb

```diff
@@ -363,6 +363,24 @@ def deferred_offer
     CandidateMailer.deferred_offer(application_form.application_choices.first)
   end
 
+  def reinstated_offer
+    application_form = FactoryBot.build_stubbed(
+      :application_form,
+      first_name: 'Harry',
+      application_choices: [
+        FactoryBot.build_stubbed(
+          :application_choice,
+          :with_accepted_offer,
+          course_option: course_option,
+          offer_deferred_at: Time.zone.local(2019, 10, 14),
+        ),
+      ],
+      candidate: candidate,
+    )
+
+    CandidateMailer.reinstated_offer(application_form.application_choices.first)
+  end
+
```

Comment:
Optional: We normally create a method for different states the email can be in. So here we would have

renistated_offer_with_conditions and reinstated_offer_without_conditions.